shared/js/background/utils.es6.js

```diff
@@ -210,7 +212,8 @@ function brokenListIndex (url, lists) {
 function getBrokenScriptLists () {
     const brokenScripts = {}
     for (const key in tdsStorage.config.features) {
-        brokenScripts[key] = tdsStorage.config.features[key]?.scripts || []
+        const featureSettings = getFeatureSettings(key)
+        brokenScripts[key] = featureSettings.scripts.map(obj => obj.domain) || []
```

Comment:
`scripts` will be undefined most times.

```suggestion
        brokenScripts[key] = featureSettings?.scripts.map(obj => obj.domain) || []
```

Reply:
Shouldn't the ? be on the `scripts` then?

Reply:
Yes sorry